### ISSUE TITLE
type errors dont highlight lines in modules

### ISSUE THREAD

**Issue by cartazio** [SublimeHaskell/SublimeHaskell#325]
I think that this is a regression, 

though clicking on the line number jumps to the correct location, so thats good

```
Build FAILED

Errors: 1, Warnings: 0, Hints: 0

Errors: 1

  /Users/carter/WorkSpace/projects/active/resin/src/Resin/Binders/Tree.hs: line 43, column 37:
    error:
    • Expecting one fewer argument to ‘IxEq f i’
      Expected kind ‘k -> *’, but ‘IxEq f i’ has kind ‘*’
    • In the type ‘IxEq f i i’
      In the definition of data constructor ‘CompactCompose’
      In the data declaration for ‘Inject’


Collected output:

In order, the following will be built (use -v for more details):
resin-0.1.0.0
Preprocessing library resin-0.1.0.0...
[5 of 5] Compiling Resin.Binders.Tree ( src/Resin/Binders/Tree.hs, /Users/carter/WorkSpace/projects/active/resin/dist-newstyle/build/resin-0.1.0.0/build/Resin/Binders/Tree.o )

```

**Comment by bscottm** [comment]
@cartazio: I'll try to reproduce. Do you have a simplified example?

**Comment by bscottm** [comment]
@cartazio: Hopefully this is fixed in 1.2.2 and you can close the issue.

**Comment by cartazio** [comment]
this is def fixed, the gui element is back
